builtin/logical/pki/path_intermediate.go

```diff
@@ -106,6 +106,13 @@ func (b *backend) pathGenerateIntermediate(
 		}
 	}
 
+	if data.Get("private_key_format").(string) == "pkcs8" {
```

Comment:
This is fine as-is, but noticed that the other params were being retrieved with `b.getGenerationParams(data)`, so this could also be moved there as well

Reply:
I went down that path originally but there lay madness. The problem is that it's not called by all functions that can generate certs and what is then done with the generation params depends on the path.

After going down that path and then one other, I realized the easiest way by far was to just transform the output once it's already generated.